sqlidetector.py

```diff
@@ -1,12 +1,21 @@
 #!/usr/bin/python3
+from sys import stdin, argv
 from core.cli import cli_opts, print_logo
 from core.app import Scanner
+from rich.console import Console
 
+console = Console(style="bold red")

```

Comment:
do we really need this style option ?

Reply:
Nope

Reply:
@knassar702

Reply:
We will merge them but kindly remove the style option

Reply:
Ok